test/unit/lib/offline-api/validation/payload-validation-field-initialvalue.spec.ts

```diff
@@ -0,0 +1,228 @@
+'use strict'
+
+import { expect } from 'chai'
+import validateBatches from './validate-batches'
+
+describe.only('payload validation (initial value)', function () {
+
+  describe('when setting initial value for non existing locales', function () {
+    it('returns an error', async function () {
+      const errors = await validateBatches(function (migration) {
+        const lunch = migration.createContentType('lunch').name('lunch')
+        lunch.createField('mainCourse')
+        .name('mainCourse')
+        .type('Symbol')
+        .initialValue({
+          'en-US': 'A Symbol',
+          'de-DE': 'A Symbol',
+          'fr-FR': 'A Symbol'
+        })
+      }, [], [], [], [ 'en-US'])
```

Comment:
All of the locale arrays have a space too much in the beginning. We really need to fix the linter setup here 😄

Reply:
Added [DANTE-102](https://contentful.atlassian.net/browse/DANTE-102) to our backlog.